packages/react-components/react-tabster/package.json

```
@@ -31,7 +31,7 @@
     "@fluentui/scripts-tasks": "*"
   },
   "dependencies": {
-    "@griffel/react": "^1.5.2",
+    "@griffel/react": "^1.5.7",
     "@fluentui/react-shared-contexts": "^9.4.0",
     "@fluentui/react-theme": "^9.1.8",
     "@fluentui/react-utilities": "^9.9.0",
```

_nit:_ Move this after the fluentui dependencies on line 37.

```suggestion
    "@fluentui/react-shared-contexts": "^9.4.0",
    "@fluentui/react-theme": "^9.1.8",
    "@fluentui/react-utilities": "^9.9.0",
    "@griffel/react": "^1.5.7",
```